apps/namadillo/src/App/Staking/Unstake.tsx

```diff
@@ -141,6 +152,9 @@ const Unstake = (): JSX.Element => {
     return "";
   })();
 
+  const unboundPeriod =
```

Comment:
Small thing: could we change instances of `unboundPeriod` to `unbondPeriod` for consistency, and rename `getUboundingPeriod` to `getUnbondingPeriod`? Might be a little more clear, matching the definition in `EpochInfo`

Reply:
ops, yes, thanks! done :)